src/postgraphile/http/createPostGraphileHttpRequestHandler.ts

```diff
@@ -221,22 +232,22 @@ export default function createPostGraphileHttpRequestHandler(options) {
 
   // We'll turn this into one function now so it can be better JIT optimised
   const bodyParserMiddlewaresComposed = bodyParserMiddlewares.reduce(
-    (parent, fn) => {
-      return (req, res, next) => {
-        parent(req, res, error => {
+    (parent: any, fn) => {
+      return (req: IncomingMessage, res: ServerResponse, next: any) => {
+        parent(req, res, (error: Error) => {
           if (error) {
             return next(error)
           }
           fn(req, res, next)
         })
       }
     },
-    (req, res, next) => next(),
+    (req: any, res: any, next: any) => next(),
```

Comment:
honestly, i dont know how this is working at all right now (before i touched it, or after).   i can't find `next` on 245  in scope?  our initial value is a function which calls `next`, which i dont see defined anywhere.

Reply:
I know how that feels. It’s one of the function arguments: req, res, NEXT 🙃

Basically this is a no-op middleware we use so we can efficiently combine the other middlewares into one middleware. It helps the JIT.

Reply:
ah, next isnt really used.  we may as well have literally used a no-op on 245.  237-241 provides a `next` implementation for us. doh!  thx :)

Reply:
Yeah this reduce is super confusing; the `next` is actually called, it's just the inner most "no-op" middleware; each middleware wraps the previous one and we have this be the initial one.